msteams-platform/messaging-extensions/api-based-microsoft-entra.md

```diff
@@ -0,0 +1,342 @@
+    5. Enter the **User consent display name**.
+    6. Enter the description for user consent description.
+    7. Select the **Enabled** option for state.
+    8. Select **Add scope**.
+
+    A message pops up on the browser stating that the scope was added.
+
+    :::image type="content" source="../assets/images/authentication/teams-sso-tabs/scope-added-msg.png" alt-text="Scope added message":::
+
+    The new scope you defined displays on the page.
+
+    :::image type="content" source="../assets/images/Copilot/api-based-me-entra-sso-scopes.png" alt-text="Screenshot shows an example of the scope added to the app in Azure portal.":::
+
+### Configure authorized client application
+
+1. Move through the **Expose an API** page to the **Authorized client application** section, and select **+ Add a client application**.
+
+    :::image type="content" source="../assets/images/authentication/teams-sso-tabs/auth-client-apps.png" alt-text="Authorized client application":::
+
+    The **Add a client application** page appears.
+
+1. Enter the appropriate Microsoft 365 client ID for the applications that you want to authorize for your app’s web application.
+
+    :::image type="content" source="../assets/images/Copilot/api-based-me-entra-sso-client-app.png" alt-text="Screenshot shows the Client ID and Authorized scopes option to add a client application to the app in Azure portal.Add a client application":::
+
+    > [!NOTE]
+    >
+    > The Microsoft 365 client IDs for mobile, desktop, and web applications for Teams are the actual IDs that you must add.
+
+    1. Select one of the following client IDs:
+
+       | Use client ID | For authorizing... |
+       | --- | --- |
+       | 1fec8e78-bce4-4aaf-ab1b-5451cc387264 | Teams mobile or desktop application |
+       | 5e3ce6c0-2b1f-4285-8d4b-75ee78787346 | Teams web application |
+
+    1. Select the application ID URI you created for your app in **Authorized scopes** to add the scope to the web API you exposed.
+
+    1. Select **Add application**.
+
+       A message pops up on the browser stating that the authorized client app was added.
+
+       :::image type="content" source="../assets/images/authentication/teams-sso-tabs/update-app-auth-msg.png" alt-text="Client application added message":::
+
+       The authorized app's client ID displays on the page.
+
+       :::image type="content" source="../assets/images/authentication/teams-sso-tabs/client-app-added.png" alt-text="Client app added and displayed":::
+
+> [!NOTE]
+> You can authorize more than one client application. Repeat the steps of this procedure for configuring another authorized client application.
+
+You've successfully configured app scope, permissions, and client applications. Ensure that you note and save the application ID URI. Next, you configure the access token version.
+
+## Update app manifest
+
+Update the following properties in the app manifest file:
```

Comment:
are we not advocating the 3p devs to use teams developer portal more ?
updating manifest manually seems more advanced user scenario to me.

Reply:
@arun2k17 , FYIP, for all the features, we request dev to update the manifest manually.